net/dbconnector.go

```diff
@@ -0,0 +1,140 @@
+package net
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	coredb "github.com/textileio/go-threads/core/db"
```

Comment:
Will make a point about something related to `net` knowing about `db` concepts later.